packages/docs/src/styles/components/colorSwatchList.scss

```diff
@@ -12,3 +12,61 @@
   vertical-align: middle;
   width: $spacer-4;
 }
+
+.c-color-swatch {
+  display: inline-block;
+  height: $spacer-4;
+  margin-bottom: $spacer-half;
+  vertical-align: middle;
+  width: $spacer-4;
+}
+
+.c-color-ramps {
+  gap: var(--grid-gutter-width);
+}
+
+.c-color-ramp__wrapper {
+  flex-basis: 256px;
+  flex-shrink: 0;
+  width: 256px;
+}
+
+.c-color-ramp {
+  clip-path: inset(0 0);
+  max-width: 256px;
+  // Move everything 1px away from the clipping rectangle to avoid a bug where
+  // we get a subpixel-tall row of transparency pattern bleeding through on the
+  // top edge of the ramp.
+  padding-top: 1px;
+  position: relative;
+}
+
+.c-color-ramp__transparency-pattern {
+  left: 0;
+  position: absolute;
+  // Move everything 1px away from the clipping rectangle to avoid a bug where
+  // we get a subpixel-tall row of transparency pattern bleeding through on the
+  // top edge of the ramp.
+  top: 1px;
+  z-index: -1;
+}
+
+.c-color-ramp__item {
+  display: flex;
+  justify-content: space-between;
+  padding: $spacer-1;
+  position: relative;
+  text-align: right;
+
+  svg {
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  code {
+    font-family: monospace;
```

Comment:
Might wanna use the same monospace font-family rule we're using for the [label mask](https://github.com/CMSgov/design-system/blob/7c6466da14cf730d030d036a4fe442bdbcef3cbc/packages/design-system/src/styles/components/_LabelMask.scss#L3)?

Reply:
Thanks for pointing it out. I don't need it anymore! I was previously selecting `span` and changed it to `code`, but `code` has its own font-family setting.